services/pkg/config/config.go

```diff
@@ -12,4 +12,7 @@ var SequencerPrivateKey = getRequiredEnvKey("SEQUENCER_PRIVATE_KEY")
 var SequencerMaxConcurrency = getOptionalEnvInt("SEQUENCER_MAX_CONCURRENCY", 200)
 var SequencerMinBatchDelayMilliseconds = getOptionalEnvInt("SEQUENCER_MIN_BATCH_DELAY_MS", 100)
 
+var SimulationProviderHTTP = getRequiredEnvString("SIMULATION_PROVIDER_URL_HTTP")
```

Comment:
Where are these new env strings set?

Reply:
they would be set wherever you are spinning up cog services ... so in for ds that would be either:

* the .devstartup.js script that helps you with a local deployment
* the helm chart that configures a kubernetes deployment
* manually on the command line if you are manually running things